cmd/server-main.go

```diff
@@ -112,6 +111,16 @@ EXAMPLES:
 `,
 }
 
+// Checks if endpoints are either available through environment
+// or command line, returns false if both fails.
+func endpointsPresent(ctx *cli.Context) bool {
+	_, ok := os.LookupEnv("MINIO_ENDPOINTS")
+	if !ok {
+		ok = ctx.Args().Present()
```

Comment:
Should this be `ok = ctx.Args().Present() || ctx.IsSet("sets")`

Reply:
There is no such thing as sets - that was a mistake.